src/views/List.jsx

```diff
@@ -22,22 +41,35 @@ export function List({ data }) {
 
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<form onSubmit={handleSubmit}>
-				<label htmlFor="search">Search:</label>
-				<input
-					type="text"
-					id="search"
-					name="search"
-					onChange={handleChange}
-					value={search}
-				/>
-				<button type="button" onClick={handleClear}>
-					x
-				</button>
-			</form>
+			{!data.length < 1 ? (
```

Comment:
In my opinon it makes more sense if this was `{data.length > 1}` but the other way seems to work as well!

Reply:
Smart. I'm not realizing that when I wrote that part, I was mixed up. Lol. This makes much more sense.